frontend/src/components/welcome/components/Info.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import styled from 'styled-components'
+import Chip from 'material-ui/Chip'
+import Avatar from 'material-ui/Avatar'
+import Typography from 'material-ui/Typography'
+
+const data = {
```

Comment:
For now is in Portuguese (we will support multilanguage soon), so the values should be: tarefas, em recompensas, usuários, and the order will be the other way around like: "20 tarefas", "$ 300 em recompensas", "30 usuários"